client/src/pages/Home/Home.tsx

```diff
@@ -1,11 +1,62 @@
 import React from "react";
-import { Container } from "./Home.styles";
+import {
+  BudgetMainContainer,
+  BudgetTitle,
+  Container,
+  ExpenseBar,
+  TotalBudgetBar,
+  Soft,
+  BudgetInfoContainer,
+  BudgetInfo,
+  BudgetLinksContainer,
+  BudgetLink,
+  PageTitle,
+  AddButton,
+} from "./Home.styles";
+
+import { BudgetData } from "../../dummydata";
+import { BsPlusSquare } from "react-icons/bs";
 
 const Home = () => {
   return (
-    <Container>
-      <h1>Home/Dashboard Page</h1>
-    </Container>
+    <>
+      <PageTitle>Your Budgets</PageTitle>
+      <Container>
+        {BudgetData.map((budget) => (
+          <BudgetMainContainer key={budget.month}>
+            <BudgetTitle>
+              {budget.month} <Soft>{budget.year}</Soft>
+            </BudgetTitle>
+
+            <TotalBudgetBar>
+              <ExpenseBar percentage={(budget.spent / budget.budget) * 100} />
+            </TotalBudgetBar>
+
+            <BudgetInfoContainer>
+              <BudgetInfo>
+                ${budget.spent} of <Soft>${budget.budget}</Soft>
+              </BudgetInfo>
+              <BudgetInfo>
+                {`${budget.budget - budget.spent < 0 ? "-" : ""}$${Math.abs(
+                  budget.budget - budget.spent
+                )}`}{" "}
```

Comment:
Is this extra space intentional?

Reply:
Looks like the Linter did that there because I'm using two javascript objects back to back?